net/stubby/files/stubby.init

```diff
@@ -23,3 +23,19 @@ start_service() {
   procd_close_instance
 }
 
+st_load_interfaces() {
+  local d
+  config_get d $1 ifname
+  [ "$1" == "wan" -o "$1" == "wan6" -o "$d" != "${d/tun}" -o "$d" != "${d/tap}" ] && ifaces=" ${1} ${ifaces}"
```

Comment:
You do not want WAN6 in here. Depending on ISP update cycle, `netifd` will generate false or weak ifup events which will cause `stubby` to restart as often as each 2 minutes. It would be better to have proper UCI (maybe LuCI) for stubby so the user can pick the interface, rather than assuming all interfaces with certain patterns. `Unbound` and `Adblock` have examples.